src/Parser.js

```diff
@@ -441,6 +466,7 @@ export default class Parser {
      */
     callFunction(
         name: string,
+        // TODO(ylem): args should be $TupleMap<ARGTYPES, ArgNode>
```

Comment:
Does `$TupleMap<ARGTYPES, ArgNode>` work in Flow?

Reply:
Unfortunately, I couldn't get it to work.